src/shared/ui/Typography/Typography.tsx

```diff
@@ -0,0 +1,81 @@
+import { ComponentPropsWithoutRef } from 'react';
+import { cva } from 'class-variance-authority';
+import cn from 'classnames';
```

Comment:
```suggestion
import cn from '@/shared/lib/core';
```
cn함수는 lib에 위치해있는 걸 사용할 수 있어요!

Reply:
반영하였습니다 감사합니다 efebc705ea77bfb243f57f7cf147b3e74a6b7b12